_data/osdf_namespace_metadata/chtc.yaml

```diff
@@ -1,15 +1,20 @@
 description: |
+  The Center for High Throughput Computing (CHTC), established in 2006, aims to bring the power of High Throughput Computing to all fields of research, and to allow the future of HTC to be shaped by insight from all fields.
 
-organization: # Name of the organization
-dataVisibility: # public, or private
-size: # In bytes so I can parse how I want
+  High Throughput Computing is a collection of principles and techniques which maximize the effective throughput of computing resources towards a given problem. When applied for scientific computing, HTC can result in improved use of a computing resource, improved automation, and help drive the scientific problem forward.
+
+The team at CHTC develops technologies and services for HTC. CHTC is the home of the HTCondor Software Suite which has over 30 years of experience in tackling HTC problems; it manages shared computing resources for researchers on the UW-Madison campus; and it leads the OSG Consortium, a national-scale environment for distributed HTC.
+
+organization: Center for High-Throughput Computing # Name of the organization
```

Comment:
We don't normally hyphenate the "high-throughput", do we?

Reply:
Good call.